src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -266,11 +256,16 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
                     readOnly: true,
                     additionalViewData: null);
 
-                content.Append(templateBuilder.Build());
-
                 if (!propertyMetadata.HideSurroundingHtml)
                 {
-                    content.AppendLine(divTag.ToHtmlContent(TagRenderMode.EndTag));
+                    var containerDivTag = new TagBuilder("div");
```

Comment:
`divTag` or perhaps `valueDivTag`; it's no more a "container" than any other `<div>` element.